internal/test/envtest/environment.go

```diff
@@ -114,19 +117,20 @@ func init() {
 	utilruntime.Must(admissionv1.AddToScheme(scheme.Scheme))
 	utilruntime.Must(runtimev1.AddToScheme(scheme.Scheme))
 	utilruntime.Must(ipamv1.AddToScheme(scheme.Scheme))
-	utilruntime.Must(builder.AddTransitionV1Beta2ToScheme(scheme.Scheme))
 	utilruntime.Must(addonsv1.AddToScheme(scheme.Scheme))
 }
 
 // RunInput is the input for Run.
 type RunInput struct {
-	M                   *testing.M
-	ManagerUncachedObjs []client.Object
-	ManagerCacheOptions cache.Options
-	SetupIndexes        func(ctx context.Context, mgr ctrl.Manager)
-	SetupReconcilers    func(ctx context.Context, mgr ctrl.Manager)
-	SetupEnv            func(e *Environment)
-	MinK8sVersion       string
+	M                           *testing.M
+	ManagerUncachedObjs         []client.Object
+	ManagerCacheOptions         cache.Options
+	SetupIndexes                func(ctx context.Context, mgr ctrl.Manager)
+	SetupReconcilers            func(ctx context.Context, mgr ctrl.Manager)
+	SetupEnv                    func(e *Environment)
+	MinK8sVersion               string
+	AdditionalSchemeBuilder     runtime.SchemeBuilder
```

Comment:
nit: should this be
```suggestion
	AdditionalSchemeBuilders     []runtime.SchemeBuilder
```
(not blocking it is an internal util, we can always iterate in future)

Reply:
runtime.Schemebuilder is already an `[]func(*Scheme) error` which is a collection of AddScheme funcs :-)